### ISSUE TITLE
grid lines aesthetic on the home page

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/www.greenwoodjs.dev#80]
## Summary
The original designs called for a grid lines aesthetic running from the header / hero banner, and as thin lines running down and behind lower sections of the page.
![Screenshot 2024-08-13 at 9 24 55 PM](https://github.com/user-attachments/assets/7be9ac18-8821-473f-8d8b-cd1aea078f68)
![Screenshot 2024-08-13 at 9 26 00 PM](https://github.com/user-attachments/assets/49b46944-6e02-4004-92f9-b5f27d38fbf5)

## Details

This is a nice to have, and found [this Reddit thread](https://www.reddit.com/r/nextjs/comments/1dbze16/how_to_show_grid_lines_like_vercel_website/) with a couple links to ways to possibly do it.

----

> Probably worth doing after #30 is merged.
